src/extension_consts.ml

```diff
@@ -56,8 +56,8 @@ module Commands = struct
 
   let open_ocamllsp_output = ocaml_prefixed "open-ocamllsp-output"
 
-  let open_ocaml_platform_ext_output =
-    ocaml_prefixed "open-ocaml-platform-ext-output"
```

Comment:
isn't this a breaking change?

Reply:
I guess if users have associated a keybinding to it, it is yes. It's a pretty niche use case I feel, but if you feel we shouldn't change the ID I'll revert this